Controller/src/main/java/org/apidb/irods/BuildEventsFile.java

```diff
@@ -26,10 +26,10 @@
  *
  * @author crisl-adm
  */
-public class ProcessIrodsEvents
+public class BuildEventsFile
```

Comment:
why did you revert to BuildEventsFile?

Reply:
I reverted it to test something and forgot to change it back again.  I will fix that now.

Reply:
Updated